README.md

```diff
@@ -1,10 +1,727 @@
-# Overview
-[The project document (i.e. latest md file)](project.v3.5.md) outlines a framework we propose grafting onto any collaborative collective. Specifically with the intent of realigning democracy for the global era. We describe a tool intended for providing a vehicle through which any collection of humans can organize themselves in any way imaginable. If you want an anarchist state, you can do that now. If you want liquid democracy where every citizen has the right to vote on every single thing to pass through the government, you can do that. If "you" want to run a dictatorship, you can make that work too I'm sure.. probably for a little while at least.
+# Overview  
```

Comment:
From a transparency angle, it is probably a good idea to note a lot of the content is rewritten by AI and iteratively shaped by a human mind. Obviously an AI can't be trusted to proofread itself..

But, on the other hand that mere fact might immediately turn off a valuable reader who would have otherwise provided valuable feedback.

The deeper divers will immediately see repo history clearly indicates the facts. Maybe that's enough =/

Might need some third party insights on this dilemma.

Reply:
in some cases, like the archetypes in this doc.. little proofreading and less iteration. iirc the deliverables in the other doc also fit this.